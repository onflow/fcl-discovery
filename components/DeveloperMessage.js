import {
  Alert,
  AlertIcon,
  AlertTitle,
  AlertDescription,
  Link,
} from '@chakra-ui/react'
import { ExternalLinkIcon } from '@chakra-ui/icons'

export default function FlowHeader() {
  return (
    <Alert status="warning" marginBottom={3}>
      <AlertIcon />
      <AlertTitle>Missing Config</AlertTitle>
      <AlertDescription>
        See how to set your app title and icon{' '}
        <Link
          href="https://github.com/onflow/fcl-discovery/blob/master/README.md#configuration"
          isExternal
        >
          here <ExternalLinkIcon mx="2px" />
        </Link>
        .
      </AlertDescription>
    </Alert>
  )
}
