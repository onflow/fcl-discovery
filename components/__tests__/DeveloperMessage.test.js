import { render } from '@testing-library/react'
import DeveloperMessage from '../DeveloperMessage'

describe('Component: DeveloperMessage', () => {
  test('should render the DeveloperMessage component', () => {
    const { container } = render(<DeveloperMessage />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
