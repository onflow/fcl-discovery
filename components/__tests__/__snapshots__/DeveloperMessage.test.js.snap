// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: DeveloperMessage should render the DeveloperMessage component 1`] = `
<div
  class="chakra-alert css-1khj809"
  role="alert"
>
  <span
    class="chakra-alert__icon css-1c6vscd"
  >
    <svg
      class="chakra-icon css-qk6lof"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M11.983,0a12.206,12.206,0,0,0-8.51,3.653A11.8,11.8,0,0,0,0,12.207,11.779,11.779,0,0,0,11.8,24h.214A12.111,12.111,0,0,0,24,11.791h0A11.766,11.766,0,0,0,11.983,0ZM10.5,16.542a1.476,1.476,0,0,1,1.449-1.53h.027a1.527,1.527,0,0,1,1.523,1.47,1.475,1.475,0,0,1-1.449,1.53h-.027A1.529,1.529,0,0,1,10.5,16.542ZM11,12.5v-6a1,1,0,0,1,2,0v6a1,1,0,1,1-2,0Z"
        fill="currentColor"
      />
    </svg>
  </span>
  <div
    class="chakra-alert__title css-0"
  >
    Missing Config
  </div>
  <div
    class="chakra-alert__desc css-6n7j50"
  >
    See how to set your app title and icon 
    <a
      class="chakra-link css-0"
      href="https://github.com/onflow/fcl-discovery/blob/master/README.md#configuration"
      rel="noopener"
      target="_blank"
    >
      here 
      <svg
        class="chakra-icon css-2y6psj"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <g
          fill="none"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-width="2"
        >
          <path
            d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"
          />
          <path
            d="M15 3h6v6"
          />
          <path
            d="M10 14L21 3"
          />
        </g>
      </svg>
    </a>
    .
  </div>
</div>
`;
